tests/acceptance/common.py

```diff
@@ -388,3 +388,26 @@ def path_restore():
     request.addfinalizer(path_restore)
 
     return os.environ['PATH']
+
+@pytest.fixture(scope="session")
+def signing_key(request):
+    subprocess.check_call(["openssl", "genrsa", "-out", "private.pem", "2048"])
```

Comment:
This is probably not changing a lot, but maybe we can have some pregenerated and hardcoded keys instead of generating new ones all the time.